### ISSUE TITLE
Feature request: Detect and ignore changes due to hard-wrapping of comments

### ISSUE THREAD

**Issue by daisylb** [Wilfred/difftastic#639]
Thanks so much for building difftastic, I use it every day and it's awesome!

One thing that I'd find really useful is if difftastic could ignore changes in comments that are there because the comment has been hard-wrapped. For instance, in the below example, difftastic highlights the text `super long` which was added, but it also highlights `contains lots`, `Like, very` and `definitely` as being deleted on each line and then inserted on the next line. It would be awesome if the latter three changes weren't highlighted, since they arose from hard-wrapping the comment.

<img width="1092" alt="Screenshot 2024-02-12 at 11 26 47" src="https://github.com/Wilfred/difftastic/assets/473960/1cee4a7c-549d-42f7-9e71-e16ccaea1359">

**Comment by Wilfred** [comment]
Yeah, I've been thinking about this too, it would be really nice to have. It's a bit fiddly, but worth implementing.
